proto/one/enterprise/data/enterprise_data_table_query.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+import "enterprise_data_table_query_parameter.proto";
+
+option csharp_namespace ="ONE.Models.CSharp.Enterprise.Data";
+
+message EnterpriseDataTableQuery {
+    // Name of DataSource which referneced by the DataTable
```

Comment:
typo on the comment, should be "referenced"

Reply:
Just installed VS Code Spell Checker extension to help me avoid this kind of spell error.  I remember I installed it before, not sure why it is gone.